test/import/FSHImporter.Alias.test.ts

```diff
@@ -67,9 +69,25 @@ describe('FSHImporter', () => {
       * code from LAINC
       `;
 
-      const result = importText(input);
+      const result = importText([new FileInfo(input)])[0];
       const rule = result.profiles.get('ObservationProfile').rules[0] as ValueSetRule;
       expect(rule.valueSet).toBe('LAINC');
     });
+
+    it('should translate an alias from any input file', () => {
+      const input = `
+      Profile: ObservationProfile
+      Parent: Observation
+      * code from LOINC
+      `;
+      const input2 = `
+      Alias: LOINC = http://loinc.org
+      `;
+
+      const results = importText([new FileInfo(input), new FileInfo(input2)]);
```

Comment:
The odd side effect mentioned at the top level can be seen here.

We can see from the passing test that `results[0]` is able to correctly resolve the alias as we need it to.

Weirdly, though, the alias is located in `results[1].aliases` and is nowhere to be found in `results[0].aliases`. This is technically not a problem, and in fact I think is objectively correctly; however, the weird result is that you technically would have no knowledge in `results[0]` that there was any alias used at all.

Do people think that's fine?

Reply:
I agree that this feels a little weird, but I think that you're right that it is objectively correct, so I think I'm okay with this as it is.

Reply:
I agree -- it is correct as implemented.